tests/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/JsonGetFieldAsTextTest.php

```diff
@@ -13,20 +14,23 @@ protected function getStringFunctions(): array
     {
         return [
             'JSON_GET_FIELD_AS_TEXT' => JsonGetFieldAsText::class,
+            'JSONB_GET_FIELD_AS_TEXT' => JsonbGetFieldAsText::class,
```

Comment:
It is cleaner when every class has a dedicated test class. Can you please bring up a new text class in the same way this was done for all other functions?